solana-generals/backend/programs-ecs/components/game/src/lib.rs

```diff
@@ -0,0 +1,157 @@
+use bolt_lang::*;
+
+declare_id!("C5iL81s4Fu6SnkQEfixFZpKPRQ32fqVizpotoLVTxA2n");
```

Comment:
I think we should't add Solana generals to the examples repo, let's leave it in it's own repository